### DISCUSSION TITLE
TE3 VS TE2 Save to folder differences?

### DISCUSSION THREAD

**Discussion by moltra** [TabularEditor/TabularEditor3#656] [🙋♀️TE3 Q&A] [answered]
My team is currently waiting for The purchasing Process to complete to get TE3.  We started using the TE3 "Save to Folder" to save changes to our DevOps server. If we use the TE2 "Save to Folder" function until we get the purchase of TE3 complete, would it mess up our Repos and changes?

**Comment by mlonsk** [comment, marked as answer]
As far as i know there should not be any difference. Tabular Editor will set the serilisation settings in your model using annotations and it both TE2 and TE3 should read it the same way.
